tests/integration/solidity_contracts/UniswapV2/test_uniswap_v2_erc20.py

```diff
@@ -86,26 +86,20 @@ async def test_should_transfer_token_when_signer_is_owner(
         async def test_should_fail_when_amount_is_greater_than_balance_and_balance_not_zero(
             self, token, owner, other
         ):
-            with pytest.raises(Exception) as e:
+            with kakarot_error():
```

Comment:
same here `assert message == "Kakarot: Reverted with reason: 147028384"` we cannot check that the returned error is relevant?

Reply:
see above